pom.xml

```diff
@@ -7,6 +7,18 @@
     <groupId>sk.uniba.fmfi</groupId>
     <artifactId>Fitness-Aplication-Schwarz</artifactId>
     <version>1.0.0-SNAPSHOT</version>
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-compiler-plugin</artifactId>
+                <configuration>
+                    <source>9</source>
+                    <target>9</target>
+                </configuration>
+            </plugin>
+        </plugins>
+    </build>
```

Comment:
Toto je závislé na Java 9, vyvýjajme to na Java 8 tak ako sme sa dohodli na začiatku. Verziu musíme mať jednotnú.

Reply:
fixed